src/Stratis.Bitcoin/Utilities/ModelStateErrors/ModelStateErrors.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Stratis.Bitcoin.Utilities.JsonErrors;
+
+namespace Stratis.Bitcoin.Utilities.ModelStateErrors
+{
+    public static  class ModelStateErrors
```

Comment:
I think this is better suited to the Stratis.Bitcoin.Features.Api, since this will only ever be used when the Api is present.